QueryEngine/HyperLogLog.h

```diff
@@ -74,6 +74,12 @@ inline void hll_unify(T1* lhs, T2* rhs, const size_t m) {
   }
 }
 
+inline int hll_sz_for_rate(const int err_percent) {
```

Comment:
Let's be more generous and use vowels: `hll_size_for_rate` or something similar.